packages/x-charts/src/PieChart/PieChart.tsx

```diff
@@ -35,15 +26,13 @@ import {
 } from '../ChartsOverlay';
 
 export interface PieChartSlots
-  extends ChartsAxisSlots,
-    PiePlotSlots,
+  extends PiePlotSlots,
```

Comment:
Do we also need to change implementation or were those props never used?

Reply:
They were used in the sense that you could update the default props to show axis

But not sure it's useful.

https://codesandbox.io/p/sandbox/quirky-voice-5s64w6?from-embed=
![image](https://github.com/user-attachments/assets/b1babf92-990b-44b1-90a5-fd77ebbbe9b8)

Reply:
My point is more that it removes the typings, but the behaviours are still the same, so the user can actually show the axes still. 

Should we also create a custom chart container for it?

Reply:
We also remove the `ChartsAxis` and `ChartsAxisHighlight` The only remaining thing is the `CartesianProvider` inside the `ResponsiveChartContainer`

From what I see it will only do a `forEach` on an empty array, so the effort of creating a custom context compared to just letting this useless context does not feel like an issue